samples/bluetooth/observer/prj.conf

```diff
@@ -1,2 +1,17 @@
 CONFIG_BT=y
 CONFIG_BT_OBSERVER=y
+
```

Comment:
instead of doing this, could we follow `peripheral_hr` that uses an alternate `prj.conf` (`prj_minimal.conf`)?
And then let the user know (in the broadcaster sample) (s)he has to use `prj_extended.conf`.
I think it's better than commented out options.

Reply:
done